sphinx/writers/latex.py

```diff
@@ -516,7 +516,7 @@ def generate(content: list[tuple[str, list[IndexEntry]]], collapsed: bool) -> No
     def _find_template(self, template_name: str) -> str:
         for template_dir in self.config.templates_path:
             # TODO: remove "_t" template suffix support after 2025-04-06
-            for template_suffix in ('_t', '.jinja'):
+            for template_suffix in ['_t', '.jinja']:
```

Comment:
```suggestion
            for template_suffix in ('_t', '.jinja'):
```

Would a tuple work here for a (tiny) memory saving?

Reply:
Yep, that works and does seem reasonable :+1: 

(I chose the `list` type after making a [mistake](https://github.com/sphinx-doc/sphinx/pull/11165#issuecomment-1499322285) while practicing removal of `_t` support; test coverage does catch that though, even if someone else makes the same mistake later)